### PR TITLE
Clean up typo

### LINKED ISSUE #124: Clean up __doc__
```
"""Description of the module"""
```
indead of
```
__doc__ = """Description of the module"""
```

### LINKED ISSUE #128: Regen pyc -> c files

### PR DESCRIPTION
**Pull request by vallsv**
Closes #124 
Closes #128

### CODE REVIEW

**Comment by vallsv** [comment]
It fixes comment from @t20100 
- remove some try-except
- clean up deg2rad into a module
- more typo
Plus
- more typo
- clean up print

**Comment by t20100** [comment]
A few additional comments:
- HiPiCimage.py: better to use logger than logging, looks like a typo.
- TiffIO.py: This is copied from PyMca, maybe you want to keep it as the original file?
- benchmark/__init__.py: The try except for import could be removed by using absolute_import and from fabio import ...
- dm3image.py l234: struct_name_length -> _struct_name_length

I think it worth it to fix the conflicts and update the cythonized files so CI actually tests the updated .pyx files.

File: fabio/openimage.py [hunk 1]
```diff
@@ -45,15 +45,12 @@
...
+from .fabioutils import FilenameObject, six, BytesIO
 from .fabioimage import FabioImage
-from . import fabioformats
 
-if six.PY2:
-    bytes = str
-    from urlparse import urlparse
-else:
-    from urllib.parse import urlparse
+# Make sure to load all foramts
```
Comment (PiRK) [inline fabio/openimage.py]: formats

**Comment by PiRK** [comment]
LGTM. I just fixed a typo in a comment (CI tests were green before that commit)

**Pull request merged**